src/live_data_server/plots/views.py

```diff
@@ -136,15 +145,46 @@ def get_data_list(_, instrument):
     instrument_object = get_object_or_404(Instrument, name=instrument.lower())
```

Comment:
This function is used by a [url that web reflectivity calls](https://github.com/neutrons/live_data_server/blob/cfe843ba3f3416474dd4707aec2f944f278a4ff7/src/live_data_server/plots/urls.py#L18). Just to make sure we don't break anything there, please add a new url and function if you need to change what the request returns